src/events/mod.rs

```diff
@@ -298,34 +298,48 @@ impl<'a> BytesEnd<'a> {
 /// Data from various events (most notably, `Event::Text`).
 #[derive(Clone, Debug)]
 pub struct BytesText<'a> {
+    // Invariant: The content is always escaped.
     content: Cow<'a, [u8]>,
 }
 
 impl<'a> BytesText<'a> {
-    /// Creates a new `BytesText` borrowing a slice
+    /// Creates a new text event from an escaped byte sequence.
```

Comment:
It's still `BytesText`, not an event.

Reply:
Whoops, I actually didn't look at the original doc comment and just rewrote it from scratch, so it only somewhat resembles what is used to look like ^^'

Will fix